util/verible-format-allowlist.txt

```diff
@@ -8,11 +8,38 @@
 hw/ip/adc_ctrl/dv/tests/adc_ctrl_test_pkg.sv
 hw/ip/adc_ctrl/dv/env/adc_ctrl_env_pkg.sv
 hw/ip/gpio/dv/tests/gpio_test_pkg.sv
+hw/dv/sv/alert_esc_agent/alert_esc_base_monitor.sv
```

Comment:
Seems like the files here are being arbitrarily chosen. Or, is there some math behind it?

Reply:
The files are hand-picked by choosing the ones where the changes
* are limited to ~100 lines to keep review manageable (larger can be revisited once we are sure the formatting is correct)
* are compared to the style guide and found compliant
So it's just manual choice at this point.